App/PyImports/ProxyPyQml.py

```diff
@@ -62,6 +63,131 @@ def loadPhasesFromFile(self):
         # explicit emit required for the view to reload the model content
         self.projectChanged.emit()
         self.onProjectUnsaved()
+    
+    @Slot(str)
+    def loadExperiment(self, selectedNameFilter):
+        """
```

Comment:
This was actually the plan in early implementations, but I moved it to the Proxy since two methods would be necessary in the Proxy regardless. In the proxy there is loadExperimentFromFile, and in addition there would need to be loadExperimentFromString, but can move the remaining logic back to ProjectSentinel.

Reply:
- Yes, I would move as much as possible from `ProxyPyQml.py` to `ProjectSentinel.py`. For example, first half of the code in `loadExperimentXYE` (up to `self._calculator_interface.setExperimentDefinitionFromString`...) as well as `get_unpolarized_cif_header` and `get_polarized_cif_header`. 
- Now, GUI loads experiment via `loadExperiment` (therefore `@Slot(str)` decorator is required). But those decorators are not needed anymore for `loadExperimentXYE` and `loadExperimentFromFile`.
- For clarity, I would rename `loadExperimentXYE` and `loadExperimentFromFile` into, e.g, `loadExperimentFromXye` and `loadExperimentFromCif`, respectively.

Reply:
The code was moved and the functions renamed as requested.